src/clr-icons/api.ts

```diff
@@ -66,22 +66,30 @@ export namespace ApiModule {
           customElements.define('clr-icon', ElementModule.ClarityIconElement);
         }
       }
-      if (icons) {
-        instance.add(icons);
-      }
     }
 
-    add(icons?: IconShapeSources): void {
-      if (typeof icons !== 'object') {
-        throw new Error(`The argument must be an object literal passed in the following pattern: 
-                  { "shape-name": "shape-template" }`);
+    add(...icons: IconShapeSources[]): void {
+      this.init();
+
+      if (!icons.length) {
+        throw new Error(
+          `The argument(s) must be an object literal passed in the following pattern: { "shape-name": "shape-template" }`
```

Comment:
This error message feels off for this case.

Honestly, I think I would just not put a guard on `icons.length` at all, `ClarityIcons.add()` with no arguments would simply not add any icons. It's silly, but I don't see a problem with it.